.github/workflows/release_mac.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [master, rel-*]
   pull_request:
-    branches: [rel-*]
+    branches: [rel-*, master]  # TODO: remove it before merge
 
```

Comment:
Mac CI fails because of this https://github.com/pypa/setuptools_scm/issues/542. Will try to reduce binary size as well.

Reply:
ONNX decided to deprecate python 3.5 in the following 1.9. Will remove it in this PR to prevent failure caused by Python 3.5 incompatibility